modules/ROOT/pages/index-secrets-manager.adoc

```diff
@@ -5,6 +5,9 @@ endif::[]
 
 Anypoint Security secrets manager uses secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
 
+[NOTE]
+If you don't see *Secrets Manager* in *Management Center* when you sign in to your Anypoint Platform account, contact your customer success manager to enable Secrets Manager for your account.
```

Comment:
This is not correct. SM is available to all customers. There is no entitlement. The explanation is complicated:
Something like -- If you cannot Access Secrets Manager or you receive an error saying 'Cannot Create Secret Groups', it is necessary to get permissions added at Access Manager. This can be added in two ways:
--If you only need access to Secrets Manager and do not manage runtime fabrics, add permissions under the Access Manager->Secrets Manager tab.
--If you are also managing runtime fabrics, it is only necessary to have the "Manage Runtime Fabric" permission added. It is not necessary to also add Secrets Manager permissions separately.